README.md

```diff
@@ -1,64 +1,23 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Decode a Morse code message
 
-# Project Name
+- Run a program using the command line.
+- Use Ruby syntax for basic programming operations.
+- Use Ruby to solve basic coding challenges.
 
-> Description the project.
 
+👤 **Bakhtiyor Yusubaliyev**
+- GitHub: [@bakhtiyorY](https://github.com/githubhandle)
+- Twitter: [@bakhtiyor-yusubali](https://twitter.com/twitterhandle)
```

Comment:
- [ ] Good job so far Team. A good project needs a good presentation. I highly recommend you make your README professional and you can achieve this by using the Microverse-provided REAMDe template. I hope that you have it and you have used it before. Then, customize it to fit your project specification. Remember that every important section should be covered 😃 👍